Sources/Document/Change/ChangeContext.swift

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+/**
+ * `ChangeContext` is used to record the context of modification when editing
+ * a document. Each time we add an operation, a new time ticket is issued.
+ * Finally returns a Change after the modification has been completed.
+ */
+class ChangeContext {
+    private let id: ChangeID
+    private let root: CRDTRoot
+    private var operations: [Operation]
+    private let message: String?
+    private var delimiter: UInt32
```

Comment:
It acts like a sequence number, Why is it called a delimiter?

Reply:
`delimiter` is used to maintain order between operations inside a Change.
Could there be a better name?
CC) @7hong13, @skhugh